sensor_msgs/Image ros_image
---
bool result
---
bool feedback